05-Token-Renewal/silent.html

```diff
@@ -21,7 +21,10 @@
   </script>
   <script>
     webAuth.parseHash(window.location.hash, function (err, response) {
-      parent.postMessage(err || response, 'http://localhost:3000');
+      parent.postMessage({
```

Comment:
you can pass `window.location.hash` directly to auth0.js instead of parsing the hash and passing the result. example: https://github.com/auth0/auth0.js/blob/master/example/callback.html

Reply:
Nice, wasn't aware of that